WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java

```diff
@@ -740,13 +738,13 @@ public void run() {
                 if (URLUtil.isNetworkUrl(mediaUrl)) {
                     String mediaId = mediaFile.getMediaId();
                     mWebView.execJavaScriptFromString("ZSSEditor.insertImage('" + mediaUrl + "', '" + mediaId + "');");
-                    AnalyticsTracker.track(Stat.EDITOR_ADDED_PHOTO_VIA_WP_MEDIA_LIBRARY);
+                    mEditorFragmentListener.onTrackableEvent(TrackableEvent.NETWORK_MEDIA_ADDED);
```

Comment:
> We should use NETWORK_IMAGE_ADDED and LOCAL_IMAGE_ADDED instead of _MEDIA_, since we're going to need to differentiate between images and video when we add video insert/upload support.

Right, that's weird I feel like I already did that change in WPAndroid